examples/asr/speech_to_label.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import hydra
+import pytorch_lightning as pl
+
+from nemo.collections.asr.models import EncDecClassificationModel
+from nemo.utils.exp_manager import exp_manager
+
+
+"""
+Basic run (on CPU for 50 epochs):
+    python examples/asr/speech_to_text.py \
```

Comment:
this command seems wrong - should be using speech_to_labels.py script, right?

Reply:
Oops forgot to update this docstring. Good catch.

Reply:
Done